Makefile

```diff
@@ -118,13 +118,35 @@ undeploy: ## Undeploy controller from the K8s cluster specified in ~/.kube/confi
 	$(KUSTOMIZE) build config/default | kubectl delete -f -
 
 
-CONTROLLER_GEN = $(shell pwd)/bin/controller-gen
-controller-gen: ## Download controller-gen locally if necessary.
-	$(call go-get-tool,$(CONTROLLER_GEN),sigs.k8s.io/controller-tools/cmd/controller-gen@v0.7.0)
+controller-gen:
+ifeq (, $(shell which controller-gen))
+		@{ \
+		set -e ;\
+		CONTROLLER_GEN_TMP_DIR=$$(mktemp -d) ;\
+		cd $$CONTROLLER_GEN_TMP_DIR ;\
+		go mod init tmp ;\
+		go install sigs.k8s.io/controller-tools/cmd/controller-gen@v0.9.2 ;\
```

Comment:
what version of the operator-sdk are you using? This bumps controller-gen to a newer version which seems to conflict with some of the other arguments. When running `make manifests` I get:

```
Error: unable to parse option "crd:maxDescLen=0,generateEmbeddedObjectMeta=false": [unknown argument "generateEmbeddedObjectMeta" (at <input>:1:40) extra arguments provided: "false" (at <input>:1:41)]
```

Reply:
Ahh, my bad, I didn't realize I was running v1.22, which is where I fixed the issue I was running into with kubebuilder previously, I'll downgrade this today and see if I can revert this change to work with v1.13